scripts/train/train.py

```diff
@@ -294,6 +294,14 @@ def main(cfg):
 
     # Build the Trainer
     print('Building trainer...')
+    save_folder = cfg.get('save_folder', None)
+    save_latest_filename = cfg.get('save_latest_filename',
+                                   'latest-rank{rank}.pt')
+    save_overwrite = cfg.get('save_overwrite', False)
+    save_weights_only = cfg.get('save_weights_only', False)
+    autoresume_default = False
+    if cfg.run_name is not None and save_folder is not None and save_latest_filename is not None and not save_overwrite and not save_weights_only:
+        autoresume_default = True
```

Comment:
Nit: can you write a comment here about turning `autoresume` on if the requirements are configured correctly?